### PR TITLE
feat: add docs on how to configure the learning goals feature

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**
## [AA-1109](https://openedx.atlassian.net/browse/AA-1109)

Add a description of your changes with links to any relevant material.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/install_operations/source/configuration/enable_weekly_learning_goals.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+***************
+
+The feature adds a new Weekly Learning Goal widget to the course home page.
+
+To use it, learners select a weekly learning goal and are automatically subscribed to goal reminder emails. The weekly reminders continue until the learner completes their course and/or their audit access ends. Learners can modify/cancel their goal and reminder settings at any time from the widget or via links in the email.
+
+For more details on this feature or screenshots see this `blog post <https://openedx.atlassian.net/wiki/spaces/PROD/blog/2021/12/07/3266805795/Course+Goal+Setting>`_.
+
+
+.. note:: This feature is only available on the new learning micro frontend. If that is not already enabled for you, you can view `instructions here <https://openedx.atlassian.net/wiki/spaces/COMM/pages/2023915819/Lilac>`_.
```
Comment (MatthewPiatetsky) [inline en_us/install_operations/source/configuration/enable_weekly_learning_goals.rst:22]: This doesn't seem like a great place to link to - is there a more authoritative config how-to for the learning mfe?

File: en_us/install_operations/source/configuration/enable_weekly_learning_goals.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+#. course_goals.number_of_days_goals
+
+#. course_experience.enable_course_goals
+
+To send goal reminder emails, you need to regularly run the following `management command <https://github.com/edx/edx-platform/blob/master/lms/djangoapps/course_goals/management/commands/goal_reminder_email.py#L101>`_.  
+
+edx.org runs this command at the following cron schedule H \*/3 \* \* \*
+
+.. note:: For the emails to work you will need to have at least one email channel configured within https://github.com/edx/edx-ace
```
Comment (MatthewPiatetsky) [inline en_us/install_operations/source/configuration/enable_weekly_learning_goals.rst:39]: is there an existing doc somewhere that walks through ace configuration? i found this doc but it doesn't actually say much on how to configure your email channel
https://github.com/edx/edx-ace/blob/master/docs/getting_started.rst#id3

for instance, i needed these settings in edx-internal
```
ACE_CHANNEL_BRAZE_CAMPAIGNS = {
	'goalreminder': '7a98f909-c8de-b6c5-e378-e2a218f4ba66:email-3529'
}
ACE_CHANNEL_DEFAULT_EMAIL = 'braze_email'
ACE_CHANNEL_TRANSACTIONAL_EMAIL = 'braze_email'
ACE_CHANNEL_BRAZE_API_KEY = '...'
ACE_CHANNEL_BRAZE_APP_ID = '...'
ACE_CHANNEL_BRAZE_FROM_EMAIL = '...'
ACE_CHANNEL_BRAZE_REST_ENDPOINT = '...'
```
this seems like the kind of thing that belongs in an ace doc, not a doc on this feature, but i'm not sure if this is documented anywhere
would it be worthwhile for me to create a ticket to add docs on how to configure ace email channels?

**Review by Dillon-Dumesnil — APPROVED** [review]
LGTM

File: en_us/install_operations/source/configuration/enable_weekly_learning_goals.rst (hunk 1 above)
Comment (Dillon-Dumesnil) [reply en_us/install_operations/source/configuration/enable_weekly_learning_goals.rst:22]: Well by the time this launches in Nutmeg, the learning MFE is the default so you might not even need this. It became the default in Maple technically and we don't promise the old views will continue to exist starting in Nutmeg.

File: en_us/install_operations/source/configuration/enable_weekly_learning_goals.rst (hunk 2 above)
Comment (Dillon-Dumesnil) [reply en_us/install_operations/source/configuration/enable_weekly_learning_goals.rst:39]: Different required settings seem to be documented in the individual channels. I think linking to the repo should be good enough.

**Pull request merged**